pkg/netflow/flowaggregator/aggregator_test.go

```diff
@@ -853,3 +855,269 @@ func TestFlowAggregator_sendExporterMetadata_singleExporterIpWithMultipleFlowTyp
 	// call sendExporterMetadata does not trigger any call to epForwarder.SendEventPlatformEventBlocking(...)
 	aggregator.sendExporterMetadata(flows, now)
 }
+
+func TestFlowAggregator_getSequenceDelta(t *testing.T) {
+	type round struct {
+		flowsToFlush          []*common.Flow
+		expectedSequenceDelta map[SequenceDeltaKey]SequenceDeltaValue
+	}
+	tests := []struct {
+		name   string
+		rounds []round
+	}{
+		{
+			name: "multiple namespaces",
+			rounds: []round{
+				{
+					flowsToFlush: []*common.Flow{
+						{
+							Namespace:    "ns1",
+							ExporterAddr: []byte{127, 0, 0, 11},
+							SequenceNum:  10,
+							FlowType:     common.TypeNetFlow5,
+						},
+						{
+							Namespace:    "ns1",
+							ExporterAddr: []byte{127, 0, 0, 11},
+							SequenceNum:  20,
+							FlowType:     common.TypeNetFlow5,
+						},
+						{
+							Namespace:    "ns2",
+							ExporterAddr: []byte{127, 0, 0, 11},
+							SequenceNum:  30,
+							FlowType:     common.TypeNetFlow5,
+						},
+					},
+					expectedSequenceDelta: map[SequenceDeltaKey]SequenceDeltaValue{
+						{FlowType: common.TypeNetFlow5, Namespace: "ns1", ExporterIP: "127.0.0.11"}: {LastSequence: 20, Delta: 20},
```

Comment:
Good catch, I forgot to implement the logic to ignore the delta when there is no previous sequence number yet.

https://github.com/DataDog/datadog-agent/pull/16819/commits/663c4cb03bd402cfdb54891767e72d554466e92c

I think this will help for cases when a devices has been running for some times and have a high sequence number.
Then later on, we start an agent to receive the flows. If we don't ignore the first sequence number, the first delta might be very high and falsely report lost flows/packets.